### PR TITLE
add artable2sql python script

### PR DESCRIPTION
**Pull request by francomor**
add airtable2sql python script

### CODE REVIEW

File: app/py_scripts/airtable2sql/airtable_to_sql.py [hunk 1]
```diff
@@ -0,0 +1,289 @@
+# pip install airtable-python-wrapper
+
+# CivicTechHub AIRTABLE TO SQL BY Franco Morero
+from airtable import Airtable
+API_KEY = ''  # your API KEY
+
+countrys_dict = {
```
Comment (omerdn1) [inline app/py_scripts/airtable2sql/airtable_to_sql.py]: We should also add a "Global" entry to this list.

Comment (francomor) [reply app/py_scripts/airtable2sql/airtable_to_sql.py]: What global entry? The API key?

Comment (omerdn1) [reply app/py_scripts/airtable2sql/airtable_to_sql.py]: @francomor  Sorry, just realized I wasn't clear. Groups can also be set to Global, meaning they're international groups. We should accommodate this need as well.

Also, should we follow the country ids from Airtable:
https://airtable.com/tblFhTf2iOcCutr9K/viwRheBZoGGuynsnr?blocks=hide

Comment (francomor) [reply app/py_scripts/airtable2sql/airtable_to_sql.py]: A okey, i traje the countri id from a sql script un the repo

Comment (geiru) [reply app/py_scripts/airtable2sql/airtable_to_sql.py]: mmh okay, not sure about this. I think generally the backend database should not rely on primary keys from airtable. 

We should store the foreign key (meaning the airtable id) for groups in the database, as a reference to detect what was already imported. 

Countries have the iso_code as a unique identifier. We should stick to the primary keys from the the sql script in the repo.

Topics do not have a foreign key, here we can detect existing topics by just comparing them. The topic name acts as a unique key.

So i think the python script should produce two files: 
1. a list of groups to insert (name, description, logo_url, country_id, airtable_group_id) with country_id being the ones from app/db/seeds/countries.sql
2. a list/dictionary of topics for each group (mapping from airtable_group_id to list of topics)

what do you think?

Comment (francomor) [reply app/py_scripts/airtable2sql/airtable_to_sql.py]: In commit @d26d206443299312a2ca96ec56443d790afc9479 i update the script, and its make the 2 files that u say

**Pull request merged**